### PR TITLE
Ticket21075 screen capture

### PR DESCRIPTION
**Pull request by damgaard22**
PR for Screen Capture

### CODE REVIEW

File: database/migrations/2019_03_13_170201_add_fields_to_devices_table3.php [hunk 1]
```diff
@@ -14,7 +14,7 @@ public function up()
 	{
 		Schema::table('devices', function(Blueprint $table)
 		{
-      $table->string('ap_name', 255)->nullable();
```
Comment (wonderfulspam) [inline database/migrations/2019_03_13_170201_add_fields_to_devices_table3.php]: The changes in this file are from a different branch, right? Doesn't seem like they should be merged.

File: resources/assets/js/admin/directives/reset.js [hunk 2]
```diff
@@ -18,16 +18,23 @@
...
+
+                            if(config['video-do-recording']) {
+                                var name = prompt($translate.instant('toast.contents.reset.video.prompt'));
+                            	Recorder.stopRecording(name);
+                            }
+			    
+			    setTimeout(function() {
                                 window.location.reload();
-                            }, 500);
+                            }, 2000);
```
Comment (wonderfulspam) [inline resources/assets/js/admin/directives/reset.js]: Is this for debugging purposes (ie. to see whether anything fails before resetting) or is this desired behaviour?

File: database/migrations/2019_03_13_170201_add_fields_to_devices_table3.php (hunk 1 above)
Comment (damgaard22) [reply database/migrations/2019_03_13_170201_add_fields_to_devices_table3.php]: Yeah should be removed from commit.

File: resources/assets/js/admin/directives/reset.js (hunk 2 above)
Comment (damgaard22) [reply resources/assets/js/admin/directives/reset.js]: Current setting of 500ms does not allow the succes/error toasts to pop up.
2000ms is the value where it doesn't feel slow and it is possible to see if the video was saved or not.

An alternative that might be to prefer is to show the toasts after the window reload.

Comment (wonderfulspam) [reply resources/assets/js/admin/directives/reset.js]: Makes sense!

**Pull request merged**